src/components/QuizCreator.vue

```diff
@@ -76,12 +76,14 @@ const handleTxtImageSrcUpdate = (src: string) => {
   const file = base64ToFile(src, fileName);
   imageFile.value = file;
   imageSrc.value = src;
+  collectPosition.value = [];
```

Comment:
not sure if this is for reset something in the editor function in the main platform, might need a comment here.

Reply:
Yes every time we upload a new image, we clear all the labels that belongs to the old image.
will add a comment